substrate/frame/revive/src/exec.rs

```diff
@@ -4607,8 +4608,8 @@ mod tests {
 				// Balance transfers should reset the output
 				ctx.ext
 					.call(
-						Weight::zero(),
-						U256::zero(),
+						Weight::MAX,
+						U256::from(u64::MAX),
```

Comment:
Same for the other occurrences.

```suggestion
						U256::MAX,
```